.github/workflows/persistent_storage.yml

```diff
@@ -50,35 +51,34 @@ jobs:
           python-version: "3.${{inputs.python3}}"
 
       - name: Install latest release
+        if: inputs.arcticdb_version == 'latest'
         run: |
           pip install arcticdb
-
-      - name: Set real storage variables
+      
+      # Currently, the oldest supported release is 3.0.0
```

Comment:
Might be good to explain what "supported" means here, ie that we're talking about forwards compatibility and we expect libs written by master only to be readable by arcticdb>=3.0.0